src/servers/Cargo.toml

```diff
@@ -51,6 +51,7 @@ futures = "0.3"
 hashbrown = "0.14"
 headers = "0.3"
 hostname = "0.3.1"
+http = "0.2.12"
```

Comment:
No. This should be visible when under "feature = testing" also.

I ever considered if it should be optional but it's actually a transitive dep for many other deps already, so I keep it as is because we do depend on http in "create servers" for any case.